code_schemes/leap_s02_rqa/leap_s02_engagement_suggestions.json

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
I think this folder should be `leap_s02_rqas` plural for consistency with `demographics`.

Note that the format we used for the csap-somalia pool was `rqas/eu_pcve/...`

Reply:
updated to `rqas/leap_s02`